scrimbot/timeoutlist.py

```diff
@@ -0,0 +1,89 @@
+from collections import namedtuple
+import discord
+import asyncio
+from datetime import datetime, timedelta, timezone
+
+
+class TimeoutList(list):
+    """List to keep track of the users in timeout. Users are represented as `NamedTuple` with id, timeout and runout_task.
+    
+    Supports:
+        `user_id in self`"""
+    User = namedtuple("User", 
+                      ["id", "timeout", "runout_task"], 
+                      defaults=[None])
+    Snowflake = namedtuple("Snowflake", "id")
+
+    def __init__(self, guild, users):
+        self.guild = guild
+        self.bot = guild.bot
+        l = []
+        for u in users:
+            task = self.bot.loop.create_task(self._timeout_runout(u["user_id"]))
+            l.append(self.User(u["user_id"],
+                               datetime.fromisoformat(u["timeout"]),
```

Comment:
All other stored times are using timestamps, would be more consistent to use those here as well.

Reply:
Alright. My idea was to make it more humanly readable, but I'll change it.